message Job {
    optional int32 tx_rate = 1;
    optional int32 duration = 2;
    optional int32 warmup = 3;
    optional int32 num_flows = 4;
    optional int32 size_min = 5;
    optional int32 size_max = 6;
    optional float life_min = 7;
    optional float life_max = 8;
    optional bool randomize = 9;
    optional bool latency = 10;
    optional bool online = 11;
    optional bool stop = 12;
    optional bool print = 13;
}
